frontend/src/metabase/collections/components/BaseItemsTable.unit.spec.js

```diff
@@ -117,6 +117,43 @@ describe("Collections BaseItemsTable", () => {
     expect(screen.queryByLabelText("Select all items")).not.toBeInTheDocument();
   });
 
+  describe("description", () => {
+    it("shows description on hover", () => {
+      const DESCRIPTION = "My collection";
+      const ITEM = getCollectionItem({ description: DESCRIPTION });
+
+      setup({ items: [ITEM] });
+
+      const icon = screen.getByRole("img", {
+        name: /info icon/i,
+      });
```

Comment:
You can use a `getIcon` helper from `__support__/ui`:

```tsx
const icon = getIcon("info");
screen.hover(icon);
```

Reply:
screen.hover is not a function. I presume you meant `userEvent.hover`

Reply:
Oh yeah, my bad! `userEvent` for sure